### PR TITLE
Fix: Fixed issue where new tab icon disappears when window is smaller than 640px

### LINKED ISSUE #11797: Bug: New tab icon disappears when window is smaller than 640 pixels
### Description

New tab icon disappears when window is smaller than 640 pixels
### Steps To Reproduce

1. Open Files
2. Reduce Files to a small window size and noticed the new tab button hides

### Requirements

Being able to click on (+) to create a new tab even when Files is not in full screen.

### Files Version

2.4.40.0

### Windows Version

10.0.22621.1413

### Log file
[debug.log](https://github.com/files-community/Files/files/11034318/debug.log)

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11797 

**Changes**
- Now `New Tab` button is always visible
- Fixes a bug where you couldn't close tabs (using `x` button) when window was smaller than 640px

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. **NOTE** To test changes you have to build Files as release or comment out line 128 of `DragZoneHelper.cs` (XAML toolbar may interfere with taps)
   2. Resize the window so that it it smaller than 640px
   3. Try to create a tab using the `+` button

### CODE REVIEW

File: src/Files.App/Actions/Navigation/NewTabAction.cs [hunk 1]
```diff
@@ -12,6 +12,8 @@ internal class NewTabAction : IAction
 
 		public HotKey HotKey { get; } = new(VirtualKey.T, VirtualKeyModifiers.Control);
 
+		public RichGlyph Glyph { get; } = new("\uE710");
```
Comment (yaira2) [inline src/Files.App/Actions/Navigation/NewTabAction.cs:15]: Is this the right icon?

Comment (ferrariofilippo) [reply src/Files.App/Actions/Navigation/NewTabAction.cs:15]: It's the same we were using before.

Comment (yaira2) [reply src/Files.App/Actions/Navigation/NewTabAction.cs:15]: We had a custom icon which was removed when I got rid of the custom icon font.

Comment (ferrariofilippo) [reply src/Files.App/Actions/Navigation/NewTabAction.cs:15]: How can I get it?

Comment (yaira2) [reply src/Files.App/Actions/Navigation/NewTabAction.cs:15]: It's not available anymore, I think we should remove this for now.

Comment (ferrariofilippo) [reply src/Files.App/Actions/Navigation/NewTabAction.cs:15]: That would remove the icon from the `TabView` control as well. I'll remove it only from the context menu, if this is what you mean

**Review by yaira2 — APPROVED** [review]
Thank you

**Pull request merged**